### PR TITLE
BaSP-TG-94: SuperAdmins redux - Get

### PR DESCRIPTION
**Pull request by aletiscornia**
- Create constants to get superAdmins.
- Create actions to get superAdmins.
- Create thunk to get superAdmins.
- Create superAdmins reducer.
- Connect list component with store to get employees.

### CODE REVIEW

File: src/Components/SuperAdmins/index.js [hunk 1]
```diff
@@ -62,23 +57,38 @@ const SuperAdmins = () => {
     { heading: 'Password', value: 'password' },
     { heading: 'Actions' }
   ];
+  if (error) {
+    return (
+      <section className={styles.container}>
+        <h2>SuperAdmins</h2>
+        <h3>{error}</h3>
+      </section>
+    );
```
Comment (MaicolFreire001) [inline src/Components/SuperAdmins/index.js:66]: You should use the messageModal here to display the error

**Review by MaicolFreire001 — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.js (hunk 1 above)
Comment (aletiscornia) [reply src/Components/SuperAdmins/index.js]: fixed

**Review by ProDIGGY-vinilos — APPROVED** [review]

**Review by fede230298 — APPROVED** [review]

**Comment by GonzaloAsencio** [comment]
Resolve Conflict

File: src/Components/SuperAdmins/index.js [hunk 2]
```diff
@@ -62,23 +57,35 @@ const SuperAdmins = () => {
     { heading: 'Password', value: 'password' },
     { heading: 'Actions' }
   ];
+  if (error) {
+    setTypeModal('Error');
+    setTextModal(error);
+    openModal();
+  }
```
Comment (MaicolFreire001) [inline src/Components/SuperAdmins/index.js:64]: This impementation will throw an infinite render loop error. You can check how you can avoid that in the other entities that were merged

**Review by MaicolFreire001 — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.js (hunk 2 above)
Comment (aletiscornia) [reply src/Components/SuperAdmins/index.js]: done

**Review by MaicolFreire001 — APPROVED** [review]

**Review by GonzaloAsencio — APPROVED** [review]

**Pull request merged**